samples/net/sockets/raw/prj.conf

```diff
@@ -0,0 +1,44 @@
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=n
+CONFIG_NET_TCP=n
+CONFIG_NET_IPV6=n
+CONFIG_NET_IPV4=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_POSIX_MAX_FDS=6
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_INIT_STACKS=y
+
+# Logging
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
+CONFIG_NET_STATISTICS=y
+CONFIG_PRINTK=y
+
+# Network buffers
+CONFIG_NET_PKT_RX_COUNT=16
+CONFIG_NET_PKT_TX_COUNT=16
+CONFIG_NET_BUF_RX_COUNT=64
+CONFIG_NET_BUF_TX_COUNT=64
+CONFIG_NET_CONTEXT_NET_PKT_POOL=y
+
+CONFIG_NET_MAX_CONTEXTS=10
+
+# Network shell
+CONFIG_NET_SHELL=y
+
+# Raw socket configuration
+CONFIG_NET_SOCKETS_RAW=y
```

Comment:
I'd say this should go to "Generic networking options". Otherwise, there you disable IPv4/IPv6, but don't "enable" anything, looks strange.